lib/src/lightwallet/fee.rs

```diff
@@ -0,0 +1,33 @@
+// Implementation of Zcash ZIP-313 that changes the default network fee with the Canopy Upgrade.
+// See https://github.com/zcash/zips/blob/2719212839358a342d5c23b4063b6bd4b5223690/zip-reduce-shielded_tx_fee.rst
+
+use zcash_primitives::consensus::{MAIN_NETWORK, NetworkUpgrade, Parameters};
+
+const PRE_CANOPY_DEFAULT_FEE: u64 = 10000;
+const POST_CANOPY_DEFAULT_FEE: u64 = 1000;
+
+// Return the default network fee at a given height. 
+pub fn get_default_fee(height: i32) -> u64 {
```

Comment:
It might be nice to eventually just replace this with [DEFAULT_FEE](https://github.com/zcash/librustzcash/blob/b1c3f9d3f033db9519d45c69d1a8e51dd6b710a5/zcash_primitives/src/transaction/components/amount.rs#L7) and continue letting this be set in one place so all consumers of librustzcash are on the same page, again. Plus, we [reduce the friction](https://github.com/zcash/zips/pull/408#issuecomment-719821561) of future changes.